apps/dg/components/case_card/case_card.js

```diff
@@ -308,16 +309,66 @@ DG.React.ready(function () {
                 tHasFormula = iAttr.get('hasFormula'),
                 tFormula = iAttr.get('formula'),
                 tCase = iShouldSummarize ? null : (iChildmostSelected && iChildmostSelected[0]) || iCases[0],
-                tValue = iShouldSummarize ? '' : tCase && tCase.getValue(tAttrID);
+                tValue = iShouldSummarize ? '' : tCase && tCase.getValue(tAttrID),
+                tType = iAttr.get('type');
             this.state.attrIndex++;
             if (isNotEmpty(tUnit))
               tUnitWithParens = ' (' + tUnit + ')';
-            if (DG.isNumeric(tValue)) {
+
+            var tColorValueField,
+                tQualitativeValueField,
+                tBoundaryValueField;
+            if (tValue instanceof Error) {
+              tValue = tValue.name + tValue.message;
+            } else if (DG.isColorSpecString(tValue)) {
+              var tColor = tinycolor( tValue.toLowerCase().replace(/\s/gi,'')),
+                  spanStyle = {
+                    backgroundColor: tColor.toString('rgb'),
+                  };
+              tColorValueField = span({
+                className: 'react-data-card-color-table-cell',
+                style: spanStyle
+              });
+            } else if (tType === 'qualitative') {
+              if (tValue === null || tValue === undefined || tValue === "") {
+                tValue = "";
+              } else {
+                var color = DG.PlotUtilities.kDefaultPointColor,
+                    tWidth = tValue;
```

Comment:
`;` => `,`, but according to my editor `tWidth` is not used so the entire line can presumably be removed.

Reply:
I had a more verbose implementation originally.  `twidth` was meant to be yanked out.